<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShSourceDock</class>
 <widget class="QDockWidget" name="ShSourceDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shader Source</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="tbExecute">
         <property name="toolTip">
          <string>Debug Shader (Ctrl+F5)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/shader-execute_32.png</normaloff>:/icons/icons/shader-execute_32.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+F5</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbReset">
         <property name="toolTip">
          <string>Reset Debug Session (Ctrl+Shift+F5)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/shader-reset_32.png</normaloff>:/icons/icons/shader-reset_32.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+F5</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbStep">
         <property name="toolTip">
          <string>Step (F11)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/shader-next_32.png</normaloff>:/icons/icons/shader-next_32.png</iconset>
         </property>
         <property name="shortcut">
          <string>F11</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbJumpOver">
         <property name="toolTip">
          <string>Step Over (F10)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/go-jump-over_32.png</normaloff>:/icons/icons/go-jump-over_32.png</iconset>
         </property>
         <property name="shortcut">
          <string>F10</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbOptions">
         <property name="toolTip">
          <string>Edit Per-Fragment Options ... (F4)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../glslDevil.qrc">
           <normaloff>:/icons/icons/shader-dbg-code_32.png</normaloff>:/icons/icons/shader-dbg-code_32.png</iconset>
         </property>
         <property name="shortcut">
          <string>F4</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="twShader">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tVertex">
       <attribute name="title">
        <string>Vertex Shader</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="teVertex">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>30</number>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tGeometry">
       <attribute name="title">
        <string>Geometry Shader</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="teGeometry">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>30</number>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tFragment">
       <attribute name="title">
        <string>Fragment Shader</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="teFragment">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>30</number>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../glslDevil.qrc"/>
 </resources>
 <connections/>
</ui>
